### DISCUSSION TITLE
Is the htmx documentation ready for translations to other languages?

### DISCUSSION THREAD

**Discussion by nazarepiedady** [bigskysoftware/htmx#674] [Ideas]
I believe that the best way to any technology do success is turning it more accessible through the process of spread its words, if possible in all developer's spoke language, that is not always the English. I use the frontend development known as **Vue.js** as an example.

I would like translate the documentation to my first spoke language, that is **Portuguese**.

**Comment by 1cg** [comment]
Hi @nazarepiedady!

That's a wonderful idea, thank you for offering.

I don't have any experience with maintaining a translation of documentation, are there any software packages for helping us do so?

I think we should wait for the 1.7 release (in December, I hope) so that the big refactor that @benpate is working in can be available.

Does that sound reasonable?

**Reply by rennerocha** [reply]
Django uses Transifex as a platform for translating its docs (https://github.com/django/django-docs-translations). It is also used by other projects. It can be a starting point to understand how your docs must be written and how to use this platform to get contributions from the community to do the translation.

**Comment by guzmanojero** [comment]
Some of the biggest projects do have different languages in their docs and that's cool. Although I think it's an enormous effort to pull that off, unless you have it translated automatically.

**Reply by rennerocha** [reply]
Automatic translations are never a good option. Computers already do a good job so you can get the whole idea of a text without problems but when we are talking about docs of a library/software, we need them to be reliable and without ambiguities. This can't be achieved with automatic translations.

**Comment by nazarepiedady** [comment]
> Hi @nazarepiedady!
> 
> That's a wonderful idea, thank you for offering.
> 
> I don't have any experience with maintaining a translation of documentation, are there any software packages for helping us do so?
> 
> I think we should wait for the 1.7 release (in December, I hope) so that the big refactor that @benpate is working in can be available.
> 
> Does that sound reasonable?

I think it is reasonable wait until after December where is planing to release the version 1.7 of the framework.

**Comment by nazarepiedady** [comment]
> Automatic translations are never a good option. Computers already do a good job so you can get the whole idea of a text without problems but when we are talking about docs of a library/software, we need them to be reliable and without ambiguities. This can't be achieved with automatic translations.

You are right, automatic translation are so cold and without live that the experience of reading any documentation translated used this approach are very painful and enigma.
